### ISSUE TITLE
Could class methods return '_' as '-' and '__' as '_'

### ISSUE THREAD

**Issue by i-void** [fazibear/opal-virtual-dom#3]
When I wrote,
`i.tiny.material_icons { 'home' }`
it renders this as `<i class="tiny material_icons">home</i>`

but as there is no `-` char accepted in method names in ruby, I think we must use convention to replace `_` to `-`, and for `_` in class names we can use `__` in method name.

so above example can be rendered as  `<i class="tiny material-icons">home</i>` beacuse underscore is not popular in class names already.

also hyperloop and clearwater uses like this.

**Comment by fazibear** [comment]
Should we really make a conversion to dash by default? It's not obvious that underscore will be converted to dash in the class property. Seen few project that class contains a lot of double underscores because of some style convention. Also, this change will break backward compatibility. 

Maybe we can use some utf8 characters. Don't know about other platform but if I press alt+'-' I get an uft8 dash which can be used as a method name.

```
✗ main ◈ 2.4.1 ➜ def asd–asd
❯   123
❯ end
:asd–asd
✓ main ◈ 2.4.1 ➜
```
What about that ?

**Comment by i-void** [comment]
I'm not sure about the normal dash and unicode dash are same characters and will work as css classes on browser side. I also tried that on my IDE and cannot succeed with a simple alt + '-' combination to get it. I copied and pasted to IDE but this time it complains about non-ascii character on method name etc... Anyway it worked at the end but I don't like it very much, may be we can discuss more about this may be we can find a better solution.
